snaps_boot/provision/hardware/pxe_utils.py

```diff
@@ -735,6 +739,32 @@ def __static_ip_configure(static_dict, proxy_dict):
     https_proxy = proxy_dict.get('https_proxy')
     iplist = []
 
+    next_word = None
+    with open("conf/pxe_cluster/ks.cfg") as openfile:
+        for line in openfile:
+            list_word = line.split()
+            for part in line.split():
+                if "rootpw" in part:
+                    next_word = list_word[list_word.index("rootpw") + 1]
+
+    user_name = 'root'
+    password = next_word
+    check_dir = os.path.isdir(consts.SSH_PATH)
+    if not check_dir:
+        os.makedirs(consts.SSH_PATH)
+    for i in range(len(host)):
```

Comment:
please change for loop to:

for this_host in host:
    target = this_host.get('access_ip')